### ISSUE TITLE
(Update) Escape from Tarkov

### ISSUE THREAD

**Issue by Vinjul1704** [Starz0r/AreWeAntiCheatYet#430]
**Game Title**
Escape from Tarkov

**Status**
Confirmed

**Proof**
As mentioned in #269, we got confirmation that the game developers contacted BattlEye to work on Proton support. The response from the game devs can be seen in this screenshot ("Nikita" in this case is the COO of the company): 
![image](https://user-images.githubusercontent.com/16061278/165561466-37e1884a-10d9-4e40-be9e-b6c3ee793d2b.png)

That screenshot was confirmed to be legitimate by a community representative and Discord head moderator of the game, as seen here:
![image](https://user-images.githubusercontent.com/16061278/165561756-12198c1b-bc6a-4323-a5e5-b2469775fa2b.png)

That screenshot comes from the "Tarkov Penguins" Discord server (https://discord.gg/5zM86yJBYs) and this is the message link: https://discord.com/channels/891628783176650773/891637148007141387/958009681119428648

You can confirm that person is indeed a community representative by simply joining the official Escape from Tarkov Discord server (https://discord.com/invite/EscapefromTarkovOfficial) and checking the user list on the right. He should be right between the walls of yellow and red names, with a dark red color.

TL;DR: The game developers asked BattlEye to work on support and we are just waiting for a BattlEye update with Tarkov support now.

**Comment by Maryxlu** [comment]
any update?

**Comment by Vinjul1704** [comment]
> any update?

I'm afraid not yet.

**Comment by Oman395** [comment]
https://store.steampowered.com/news/group/4145017/view/3104663180636096966?l=english

Apparently BE actually works with proton, and iirc BSG is working on a steam release for tarkov, so we might not be waiting too much longer?

**Comment by Vinjul1704** [comment]
> https://store.steampowered.com/news/group/4145017/view/3104663180636096966?l=english
> 
> Apparently BE actually works with proton, and iirc BSG is working on a steam release for tarkov, so we might not be waiting too much longer?

If you scroll up, you can see that it was already confirmed that they will update the BE runtime for Tarkov in the future. Proton and the game being on Steam are not necessary, as the patches to load it are designed to work with other wine versions outside of Steam as well.

**Comment by forgondolin** [comment]
Super pumped for this, can't wait to play online raids

**Comment by gunererd** [comment]
come on already!

**Comment by BadKiko** [comment]
still wait!

**Comment by LtdJorge** [comment]
@Starz0r this should not have been closed, right? It was marked as Confirmed, but right now the stat is set as Planned, which is the correct one.

**Comment by nyuware** [comment]
Tarkov just wiped recently, any update on this ?

**Comment by SkippTekk** [comment]
Since this hasn't been applied, it shouldn't be closed.... PLEASE have this done asap properly... the Linux community have been waiting patiently.

**Comment by mylesw42** [comment]
Bumping, this should not be closed.

**Comment by Starz0r** [comment]
Not really sure what is being asked here? There doesn't look like there has been any movement on this front, so I'm really contemplating if I should just move this back into the "Broken" state. If it really is still planned by the developers to enable Linux support then I'll keep it marked the way it is currently on the website. Why this issue would need to be reopened is beyond me, nothing has happened yet. Either way, if something DOES happen, feel free to open a new issue with the more information.
	
Locked.
